src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -48,8 +49,16 @@ namespace OpenMS
 
     /**
       @brief Integrate intensities in a spectrum in range @p im_range (if defined) for multiple windows.
-    */
+      @p windows_center is a vector of the center location of the windows.
+      @p width is the width of the windows across mz
+      @p RangeMobility is the range of the IM dimension (if defined)
```

Comment:
```suggestion
      @param im_range The range of the IM dimension (if defined)
```

wrong variable name...